corehq/apps/app_manager/id_strings.py

```diff
@@ -90,6 +90,15 @@ def detail_column_header_locale(module, detail_type, column):
     )
 
 
+@pattern('m%d.%s.%s.display')
+def sort_only_column_header_locale(module, detail_type, sort_element):
+    return u"m{module.id}.{detail_type}.{field}.display".format(
+        detail_type=detail_type,
+        module=module,
+        d=sort_element,
```

Comment:
What is `d` for? I don't see it in the format string. Also this should include something to indicate that it's for a sort only field. e.g. `"{module.id}.{detail_type}.{d.model}_{field}.sort_display"`